Orphanage/README.md

```diff
@@ -0,0 +1,41 @@
+# Orphanage
+DOM Element highlighter tool, updates role attributes of document elements.
+
+<h3>Project scope:</h3> 
```

Comment:
Use proper markdown syntax :-) Also, shouldn't this be an H2?

https://guides.github.com/features/mastering-markdown/